tests/calculators/test_torsion_calculators.py

```diff
@@ -0,0 +1,47 @@
+from pytest import approx
+import stko
+import stk
+
+
+def test_torsion_matching():
+    """confirm torsions are appropriately mapped from constructed
+    molecules to building blocks
+    """
+
+    def test_polymer(bb1: stk.BuildingBlock, bb2: stk.BuildingBlock):
```

Comment:
I feel like this should be a private function outside of this function (`def _test_polymer`)

Reply:
I would also just call it `polymer` because it is not a test.

Reply:
Actually something like `_get_polymer_angles` is a better name.